assets/app/view/game/route_selector.rb

```diff
@@ -183,23 +189,35 @@ def render
           },
         }
 
-        instructions = 'Click revenue centers, again to cycle paths.'
-        instructions += ' Click button under Revenue to pick number of halts.' if render_halts
+        if @step.respond_to?(:instructions)
+          instructions = @step.instructions
+        else
+          instructions = 'Click revenue centers, again to cycle paths.'
+          instructions += ' Click button under Revenue to pick number of halts.' if render_halts
+        end
+
+        h3_text = @step.respond_to?(:section_header) ? @step.section_header : 'Select Routes'
+
+        table_header = if @step.respond_to?(:show_table_header?) && !@step.show_table_header?
```

Comment:
why

Reply:
Because the header is confusing for track conversion.